docs/source/interfaces/DrILL.rst

```diff
@@ -194,6 +194,25 @@ row(s). To get further information about the errors, one has to look into the
 Mantid logs.
 
 
+Automatic data export
+---------------------
+
+After the processing, the export of reduced data is automatically triggered. The
+configuration of this mechanism is done via the export dialog that can be opened
+from the tool bar.
+
+.. image:: ../images/drill-export.png
+   :align: center
+
+For each acquision mode, a list of adapated algorithms will be displayed in that
```

Comment:
I think this should be "acquisition mode"

Reply:
Done.